scripts/test_emod3d.sh

```diff
@@ -32,13 +42,47 @@ do
 #            echo "rlog =1"
     else
         rlog_check=1
-#        echo "$rlog not finsihed"
+        echo "$rlog not finished"
         break
+    fi
+
+    rootFile="../OutBin/${rlog/%.rlog/.e3d}"
+
+    seisFile=`echo $rootFile | sed 's/\(.*\)-/\1_seis-/'`
+    xytsFile=`echo $rootFile | sed 's/\(.*\)-/\1_xyts-/'`
 
+    #Check that if the seis or xyts file exists for a certain integer then the other also exists
+    if [[ -f "$seisFile" ]] && [[ ! -f "$xytsFile" ]];
+    then
+        echo "Found the seis file $seisFile, but didn't find the matching xyts file $xytsFile"
+        rlog_check=1
+        break
+    fi
+    if [[ ! -f "$seisFile" ]] && [[ -f "$xytsFile" ]];
+    then
+        echo "Found the xyts file $xytsFile, but didn't find the matching seis file $seisFile"
+        rlog_check=1
+        break
     fi
 done
 
+# Check the integrity of the seisfiles only if all files that are expected to be there are present
+# This is done by attempting to load them into LFSeis which will check them
 if [[ $rlog_check == 0 ]];
+then
+    python3 -c "from qcore import timeseries; timeseries.LFSeis('../OutBin');" 2>/dev/null
+    seisIntegrity=$?
+    if [[ $seisIntegrity != 0 ]];
+    then
+        echo "At least one file failed the integrity check"
+    fi
+fi
+
+# EMOD3D is not considered to be completed if:
+#   Any Rlog does not have IS FINISHED
+#   There is a seis file without an equally indexed xyts file or vice versa
+#   Any seis file is unable to be loaded by LFSeis
```

Comment:
How performant is the loading? Aka for large seis files does this take a while?

Reply:
for 400m AlpineF2K it usually take around 20~30 sec to load.